sympy/assumptions/handlers/order.py

```diff
@@ -31,10 +31,10 @@ def Expr(expr, assumptions):
     @staticmethod
     def _number(expr, assumptions):
         r, i = expr.as_real_imag()
-        # If the imaginary part can symbolically be shown to be zero then
-        # we just evaluate the real part; otherwise we evaluate the imaginary
-        # part to see if it actually evaluates to zero and if it does then
-        # we make the comparison between the real part and zero.
+        """If the imaginary part can symbolically be shown to be zero then
```

Comment:
Don't use strings for comments.

Reply:
Okay sir. @ShubhamKJha Can you make the change here? It will make you the co-author.